tahoe_idp/magiclink_settings.py

```diff
@@ -0,0 +1,80 @@
+# flake8: noqa: E501
+import warnings
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
+LOGIN_FAILED_TEMPLATE_NAME = getattr(
+    settings,
+    'MAGICLINK_LOGIN_FAILED_TEMPLATE_NAME',
+    'tahoe_idp/magiclink_login_failed.html'
+)
```

Comment:
Let's not use the template and use the `LOGIN_FAILED_REDIRECT`.

A good value for it in `server-vars.yml` is `/500`, see https://studio.tahoe.appsembler.com/500



```suggestion
```